lib/fast_cache/cache.rb

```diff
@@ -84,6 +85,7 @@ def delete(key)
       entry = @data.delete(key)
       if entry
         @expires_at.delete(entry)
+        @cleanup.call(entry.value) if @cleanup
```

Comment:
Or, we could not call the cleanup block here?  That would seem inconsistent to me, though.